command/generate_test.go

```diff
@@ -106,25 +106,6 @@ func (s *generateSuite) TestExecuteSuccess() {
 	s.NoError(c.Execute(s.ctx, s.comm, s.logger, s.conf))
 }
 
-func (s *generateSuite) TestExecuteSuccessWithFilesFilter() {
```

Comment:
Does this test fail? Why?

Also, you should add a globbing and an expansion test, if you can.

Reply:
Well it doesn't fail, it just sort of doesn't exist anymore, since this test specifically uses filesFilter. Other than that it's exactly the same as TestExecuteSuccess.